academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,170 @@
+* Interchain Accounts - [ICS 27](https://github.com/cosmos/ibc/tree/master/spec/app/ics-027-interchain-accounts)
+  ICS-27 Interchain Accounts outlines a cross-chain account management protocol built on IBC. ICS-27-enabled chains can programmatically create accounts on other ICS-27-enabled chains and control these accounts via IBC transactions, instead of having to sign with a private key. Interchain Accounts contain all of the capabilities of a normal account (i.e. stake, send, vote) but instead are managed by a separate chain via IBC in a way such that the owner account on the controller chain retains full control over any interchain account(s) it registers on host chain(s).
+* General relayer incentivization mechanism - [ICS 29](https://github.com/cosmos/ibc/tree/master/spec/app/ics-029-fee-payment)
+  Incentivize relayers for scaling.
+* IBC application middleware - [ICS 30](https://github.com/cosmos/ibc/tree/master/spec/app/ics-030-middleware)
+  The middleware allows developers to define the extensions as separate modules that can wrap over the base application.
+
+<HighlightBox type="info">
+
+This list can be and will be extended with time. New concepts such as Interchain Accounts will continue to further decrease adoption times and provide new possibilities to what IBC offers - a blockchain can control an account on another blockchain with Interchain Accounts.
+
+</HighlightBox>
+
+## Custom application
+
+The interface exposed by the standard IBC implementation is referred to as the **[IBC handler](https://github.com/cosmos/ibc/tree/master/spec/core/ics-025-handler-interface)**. Modules can use the IBC handler for client, connection, and channel lifecycle management, and to send and receive packets.
+
+![Packet flow](images/packetflow.png)
+
+A packet is first received by the handler of the application module before getting passed down to callbacks on the IBC core handler. In addition, an acknowledgment is sent back to the packet sender.
+
+![IBC handler](images/ibchandler.png)
+
+For each connection between two blockchains, different channels can be established. **Channels** provide the (possibly ordered) exactly-once delivery of packets and handle module permissioning through **ports**. A **module** is bound to a port, and channels are end-to-end between two ports.
+
+## Implementations
+
+IBC is not just a specification in a whitepaper, it is used for hundreds of thousands of transactions per day across various chains.
+
+<HighlightBox type="tip">
+
+Have a look at the [Map of Zones](https://mapofzones.com/?testnet=false&period=24&tableOrderBy=totalIbcTxs&tableOrderSort=desc) to get an overview of blockchains with IBC implementation. In it, you can find blockchains like the Cosmos Hub or Osmosis and see how they are connected to a lot of other blockchains - they are used by many applications.
+
+</HighlightBox>
+
+Some **IBC-related implementations** to highlight are:
```

Comment:
The below are all quite different types of project, not sure if listing them under IBC-related implementations makes sense here. Although I can see that pointing out that the whole IBC ecosystem depends on the development of different building blocks (IBC core implementations, applications, relayer software etc.) working together in synergy might be good

Reply:
agree the implementation list doesnt really make sense as a random selection. let's discuss in the call.